demo/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,238 @@
+
+> Alfred decides to copy over the code of a component he found, and implement the feature himself, and as a result now has to maintain the entire component as well, which becomes an added burden to the project.
+
+### Accessibility
+
+Your components should be usable by every kind of user. Accessibility is hard to get right, but an essential feature, additionally, accessibility issues can be impossible to fix at a later stage without breaking changes, so it is crucial to incorporate accessibility right from the start.
+
+> Somewhere down the line, Alfred finds an accessibility issue with a component he found online. He can't fix this issue in his code, so he asks the maintainers to help him out. They tell him they can't fix it because they don't want to do a breaking change.
+
+Or, alternatively
+
+> Alfred writes his own component, and the accessibility issue is hard to fix because of the way he initially wrote his HTML structure, and will also lead to a breaking change.
+
+### Summary
+
+Does Alfred's story feel relatable?
+
+It highlights multiple common issues in modern web development:
+- Finding and adding a dependency is a big deal
+- Picking something because of looks alone might not always be the best choice
+- Adjusting behavior or styling can be tough to maintain
+- If a package is popular it doesn't mean it has good accessibility or performance
+
+### What can we do to tackle these issues?
+
+🤔 Imagine having components that purely focus on functionality, and leave the design up to you  
+🤔 Imagine that these components come with great accessibility and performance  
+🤔 Imagine these components are highly extendable and flexible  
+
+Now stop imagining, and look at Lion 🦁
+
+## What is Lion?
+
+> We want to bring the web forward - one component at a time.
+
+Lion is a white-label, open-source, framework-agnostic component library, and can be the foundation for your codified in-house Design System. Aligning on design and usability is already a tough challenge, with Lion, we hope to take some of the work out of your hands.
+
+### White Label
+Lion is a core package of white label Web Components. What this means is that the core components only consist of very minimal styling, yet has all the fundamental functionalities. White label products are often created so others can easily rebrand them with their own visual identity. This is great because it means that everyone can use the core functionalities of our components while bringing their own branding or Design System because surprisingly, not everyone loves orange.
+
+And this is exactly what we at ING do as well. Our very own ing-web components extend the Lion components and apply our own ING visual identity which mostly is a thin layer on top of Lion.
+
+Check out the [Lion demo's](http://lion-web-components.netlify.com/). Looks plain, doesn't it? And now compare Lion to ing-web:
```

Comment:
The lion demo is missing the `input-range`

Reply:
care to add it? 😬

Reply:
https://github.com/ing-bank/lion/pull/543

The lion umbrella form also contains a `select-rich`, you might want to disable it before creating a screen-shot.